app/javascript/google_analytics.js.erb

```diff
@@ -0,0 +1,16 @@
+document.addEventListener('turbolinks:load', function(event) {
+  if (typeof gtag === 'function') {
+    gtag('config', 'G-XH7G86BZJB', {
+      'page_location': event.data.url
+      'page_path': event.srcElement.location.pathname,
+      'page_title': event.srcElement.title
+    })
+  }
+  if (typeof gtag_stage === 'function') {
+    gtag_stage('config', 'G-HP4X05RVXV', {
+      'page_location': event.data.url
+      'page_path': event.srcElement.location.pathname,
+      'page_title': event.srcElement.title
+    })
+  }
+})
```

Comment:
Does this need to be `.js.erb` if there's no embedded Ruby here? Would it work equally well as `.js`?

Reply:
In the beginning there was some ruby in it but I removed it. We might add some ruby in it when the credentials are stored on the production and staging servers.